### PR TITLE
feat: created the responsive image component.

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
- added the square images 
- added the circle images
- made the images responsive

image-preview
e

### CODE REVIEW

File: components/image.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    background-color: hsl(var(--primary-lighter));
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 2rem;
+    padding: 1rem;
+}
+
+.image{
+    width: 100%;
```
Comment (rahulcs754) [inline components/image.css]: use max-width not width and also read about why you will use max-width

**Review by rahulcs754 — COMMENTED** [review]
I looked your code that is nice and please  make suggested changed

File: components/image.css (hunk 1 above)
Comment (vaibhav-mahalle) [reply components/image.css]: made the suggested changes. please look into it.

**Review by rahulcs754 — APPROVED** [review]

File: components/image.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+@import url(/stylesheets/main.css);
+
+.image-container{
+    background-color: hsl(var(--primary-lighter));
+    display: flex;
+    align-items: center;
+    justify-content: center;
```
Comment (nakulsharma15) [inline components/image.css:7]: You can extract this in a utility class so that it can be used anywhere else as well

File: components/image.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    gap: 2rem;
+    padding: 1rem;
+}
+
+.image{
+    width: max-width;
```
Comment (nakulsharma15) [inline components/image.css]: I think you're trying to give max-width to image. The syntax of max-width is: "max-width: value;".
 e.g., "max-width: 5rem;"

**Review by nakulsharma15 — COMMENTED** [review]
Suggested a few changes

File: components/image.css (hunk 2 above)
Comment (vaibhav-mahalle) [reply components/image.css:7]: i will do it when working on utility class

File: components/image.css (hunk 3 above)
Comment (vaibhav-mahalle) [reply components/image.css]: made the necessary changes.

**Review by nakulsharma15 — APPROVED** [review]

**Pull request merged**